rfc8446.xml

```diff
@@ -208,11 +197,10 @@ ChangeCipherSpec (except when needed for middlebox compatibility).</t>
   <t>Elliptic curve algorithms are now in the base spec, and new signature
 algorithms, such as ed25519 and ed448, are included. TLS 1.3 removed point format
 negotiation in favor of a single point format for each curve.</t>
-  <t>Other cryptographic improvements have been added, including the removal
-of compression and custom Ephemeral Diffie-Hellman (DHE) groups, changing the
-RSA padding to use the RSA Probabilistic Signature Scheme (RSASSA-PSS), and
-the removal of DSA.
-
+  <t>Other cryptographic improvements, including the removal
+of compression, the Digital Signature Algorithm (DSA), and custom Ephemeral
+Diffie-Hellman (DHE) groups, changing the RSA padding to use the RSA
+Probabilistic Signature Scheme (RSASSA-PSS), have been added.
```

Comment:
With the reordering here, we now need an "and" before "changing the RSA padding".

Reply:
I'd also consider changing "added" to be "made".